README.md

```diff
@@ -91,6 +92,32 @@ is unset (those tests are not run). To run all tests, execute
 TEST_WITH_NETWORK=1 make test
 ```
 
+## Docker
+
+Build a local base image:
+
+```sh
+make docker-build
+```
+
+Tag the local base image with the current version number:
+
+```sh
+make docker-tag-version
+```
+
+Tag the local base image as the latest version:
+
+```sh
+make docker-tag-latest
+```
```

Comment:
The rule also builds an images. I suggest:

"Build an image tagged with the 'latest':"

Reply:
I changed the implementation to match the documentation.